cmds/core/dhclient/dhclient.go

```diff
@@ -47,51 +47,79 @@ var (
 	v4Port = flag.Int("v4-port", dhcpv4.ServerPort, "DHCPv4 server port to send to")
 )
 
-func main() {
-	flag.Parse()
-	if len(flag.Args()) > 1 {
-		log.Fatalf("only one re")
-	}
+type Opts struct {
```

Comment:
does this struct need to be exported? All the fields are unexported so maybe it makes sense to make this struct unexported too?

Reply:
I agree, this does not need to be exported at all. I will fix it up